### PR TITLE
Clean up ViewPumpLayoutInflater

### PR DESCRIPTION
**Pull request by ZacSweers**
Style fixes and also removes the unused constructor

### CODE REVIEW

File: viewpump/src/main/java/io/github/inflationx/viewpump/internal/-ViewPumpLayoutInflater.kt [hunk 1]
```diff
@@ -14,32 +14,21 @@ import org.xmlpull.v1.XmlPullParser
 import java.lang.reflect.Field
 
 @Suppress("ClassName")
-internal class `-ViewPumpLayoutInflater` : LayoutInflater, `-ViewPumpActivityFactory` {
+internal class `-ViewPumpLayoutInflater`(
+    original: LayoutInflater,
+    newContext: Context,
+    cloned: Boolean
+) : LayoutInflater(original, newContext), `-ViewPumpActivityFactory` {
```
Comment (chrisjenx) [inline viewpump/src/main/java/io/github/inflationx/viewpump/internal/-ViewPumpLayoutInflater.kt]: -ViewPump what's the - for?

Comment (ZacSweers) [reply viewpump/src/main/java/io/github/inflationx/viewpump/internal/-ViewPumpLayoutInflater.kt]: It's a clever trick that hides it from Java. Classes with leading chars that aren't valid in Java (but are valid in the JVM) just won't index if someone tries to use it

Comment (chrisjenx) [reply viewpump/src/main/java/io/github/inflationx/viewpump/internal/-ViewPumpLayoutInflater.kt]: Oh and still valid name for the AndroidVM?

Comment (ZacSweers) [reply viewpump/src/main/java/io/github/inflationx/viewpump/internal/-ViewPumpLayoutInflater.kt]: Yep! Okio does the same approach

**Pull request merged**